ecs_pod_creator.go

```diff
@@ -454,6 +454,12 @@ func (s *SecretOptions) SetExists(exists bool) *SecretOptions {
 	return s
 }
 
+// SetOwned returns whether or not the secret is owned by its pod.
```

Comment:
I think the documentation is wrong here.

Reply:
This is accurate. The secret options in this case are referring to a secret in Secrets Manager, which is either dependent or independent of the pod's lifecycle depending on this flag.